src/lib/utils/hooks/SynapseAPI/file/useFiles.ts

```diff
@@ -32,3 +32,31 @@ export function useGetPresignedUrlContent(
     },
   )
 }
+
+export function useGetProfileImage(
+  userId: string,
+  options?: Omit<UseQueryOptions<Blob | null, SynapseClientError>, 'staleTime'>,
+) {
+  const queryFn = async () => {
+    const presignedUrl = await SynapseClient.getProfilePicPreviewPresignedUrl(
+      userId,
+    )
+    if (presignedUrl) {
+      const data = await fetch(presignedUrl, {
```

Comment:
```suggestion
      // Fetch the presigned URL right away because it will expire
      const data = await fetch(presignedUrl, {
```

Reply:
Nice comment, recommend this is committed.